test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -642,6 +642,20 @@ public void ItMigratesAndBuildsLibrary(string projectName)
         }
 
         [Fact]
+        public void ItMigratesAndBuildsAppWithExplicitInclude()
```

Comment:
I feel like we should be able to get away here with a UnitTest instead of an E2E test.